sys/reaper.go

```diff
@@ -34,16 +34,16 @@ func Reap(wait bool) (exits []Exit, err error) {
 		}
 		exits = append(exits, Exit{
 			Pid:    pid,
-			Status: exitStatus(ws),
+			Status: ExitStatus(ws),
 		})
 	}
 }
 
 const exitSignalOffset = 128
 
-// exitStatus returns the correct exit status for a process based on if it
+// ExitStatus returns the correct exit status for a process based on if it
 // was signaled or exited cleanly
-func exitStatus(status syscall.WaitStatus) int {
+func ExitStatus(status syscall.WaitStatus) int {
```

Comment:
Let's not export this if it isn't used anywhere. If we need it later we can export it. Plus we should avoid exporting functions which have syscall arguments if it can be avoided.